java/server/src/org/openqa/selenium/remote/server/log/TerseFormatter.java

```diff
@@ -40,6 +40,13 @@
    */
   private static final String SUFFIX = " - ";
 
+  /**
+   * Line separator string.  This is the value of the line.separator
+   * property at the moment that the TerseFormatter was created.
+   */
+  private final String lineSeparator = java.security.AccessController.doPrivileged(
+      new sun.security.action.GetPropertyAction("line.separator"));
```

Comment:
Noted. I'll fix it in a bit.

I took this line from the reference implementation in the JDK itself. I didn't realize though that it uses internal API.